yuniql-platforms/postgresql/PostgreSqlBulkImportService.cs

```diff
@@ -1,277 +1,277 @@
-﻿using System.Data;
-using System.IO;
-using Yuniql.Extensibility;
-using NpgsqlTypes;
-using System.Collections.Generic;
-using Npgsql;
-using System;
-using System.Linq;
-using Yuniql.Extensibility.BulkCsvParser;
+﻿using System.Data;
+using System.IO;
+using Yuniql.Extensibility;
+using NpgsqlTypes;
+using System.Collections.Generic;
+using Npgsql;
+using System;
+using System.Linq;
+using Yuniql.Extensibility.BulkCsvParser;
 using System.Diagnostics;
 
 //https://github.com/22222/CsvTextFieldParser
-namespace Yuniql.PostgreSql
-{
-    ///<inheritdoc/>
-    public class PostgreSqlBulkImportService : IBulkImportService
-    {
-        private string _connectionString;
+namespace Yuniql.PostgreSql

```

Comment:
Might be a mistake that this file change?

Reply:
yes I didn't mean to change it, I will checkout this file to master 👍